src/Compiler/CompilerUtils.cpp

```diff
@@ -372,10 +392,9 @@ static std::string genSharedLib(string outputBaseName, std::vector<string> opts,
 #ifdef _WIN32
   string sharedLibPath = outputBaseName + ".dll";
   std::vector<string> outputOpt = {"/Fe:" + sharedLibPath};
-  // link has to be before def and libpath since they need to be passed through
-  // to the linker
-  std::vector<string> sharedLibOpts = {
-      "/LD", "/link", "/NOLOGO", "/def:" + outputBaseName + ".def"};
```

Comment:
If we remove the .def file, the functions exported from cruntimelib also need to be marked as dll exports because they can no longer be exported in any other way

Reply:
Done. Thank you!